marimo/_sql/parse.py

```diff
@@ -98,11 +98,18 @@ def _parse_sql_duckdb(
     - Invalid function names do not throw errors
     - Some syntax errors do not throw errors since they are not errors in the AST parser
     """
-    DependencyManager.duckdb.require("to parse sql")
+    if not DependencyManager.duckdb.has():
+        return None, "DuckDB not installed"
 
     import duckdb
 
-    relation = duckdb.execute(JSON_SERIALIZE_QUERY, [query])
+    json_serialize_query = (
+        JSON_SERIALIZE_QUERY
+        if duckdb.__version__ >= "1.1.0"
```

Comment:
does this check actually work? maybe we can remove it by the time we get to "1.11.0"